src/components/Deposits.js

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import Title from "./Title";
+import { DesktopDatePicker } from "@mui/x-date-pickers/DesktopDatePicker";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { TextField } from "@mui/material";
+import { Grid } from "@mui/material";
+
+function preventDefault(event) {
+  event.preventDefault();
+}
+
+export default function Deposits() {
+  const [startValue, setStartValue] = React.useState(new Date("2005-04-02"));
+  const [endValue, setEndValue] = React.useState(
+    new Date().toISOString().slice(0, 10)
+  );
+
+  const handleStartChange = (newValue) => {
+    setStartValue(newValue);
+  };
+
+  const handleEndChange = (newValue) => {
+    setEndValue(newValue);
+  };
+
+  return (
+    <React.Fragment>
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
+        <Grid container spacing={2}>
+          <Grid item xs={12}>
+            <Typography component="p" variant="subtitle1">
+              Ustaw datę początkową
+            </Typography>
+          </Grid>
+          <Grid item xs={12}>
+            <DesktopDatePicker
+              label="Date desktop"
+              inputFormat="MM/dd/yyyy"
```

Comment:
Ogolnie to hujowo pomyslane bo dane moga byc nadupcane co 10s i wtedy chcemu z danej minuty se obczaic